src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -2829,41 +2835,23 @@ public static int indexOfAny(final CharSequence cs, final String searchChars) {
 
      * </pre>
      *
-     * @param cs  the CharSequence to check, may be null
+     * @param srcChars  the CharSequence to check, may be null
```

Comment:
Don't rename parameters in methods, it makes the PR noisy and makes it harder to review.